README.md

```diff
@@ -234,6 +234,28 @@ While you’re setting things up consider setting the config options `async:
 false` and `log_only: true` to take ActiveJob and BigQuery (respectively) out
 of the loop.
 
+### 9. Web request event user identifier
+
+#### User identifier
+
+All web request events will add a `user_id` to the event data sent to BigQuery. The `user_id` will only be populated if the controller defines a `current_user` method and it responds to `id`, otherwise the `user_id` will be `nil`.
+
+#### Custom User Identifier
+
+If a field other than `id` is required for the user identifier, then  a custom user identifier proc can be defined in `config/initializers/dfe_analytics.rb`:
+
+```ruby
+DfE::Analytics.config.user_idenitifier = proc { |user| user&.id }
+```
+
+#### User ID anonymisation
+
+The `user_id` in the web request event will not be anonymised by default. This can be changed by updating  the configuration option in `config/initializers/dfe_analytics.rb`:
```

Comment:
@asatwal would it be worth adding a line here like 'You probably want to anonymise user_id if the field it comes from in your schema is in analytics_pii.yml so that analysts can join the IDs together. If your user_id is not in analytics_pii.yml but is in analytics.yml then you probably want to ensure user_id is *not* anonymised so you can join the IDs together.'

Reply:
Good idea. Done.